content/special-topics/variables.md

```diff
@@ -119,7 +119,7 @@ $(JOBCODE)                 internal jobcode of current job
 
 # metadata
 
-Fields from the [metadata editor](../module-reference/utility-modules/shared/metadata-editor.md) can be used for variable substitution using the last part of the associated tag name as the variable name. For example, the xmp tag `Xmp.iptc.location` is available as the variable `$(LOCATION)`.
+Fields from the [metadata editor](../module-reference/utility-modules/shared/metadata-editor.md) can be used for variable substitution using the full case sensitive Xmp tag name as the variable name, i.e. `Xmp.iptc.location`.
```

Comment:
same here

Reply:
Understood, I don't think it's the native language of most contributors tbh. This is what copy-editing is for :)